core/finality-grandpa/src/lib.rs

```diff
@@ -52,6 +52,9 @@
 //! or prune any signaled changes based on whether the signaling block is
 //! included in the newly-finalized chain.
 #![forbid(warnings)]
+/// FIXME
```

Comment:
I'll happily put this into a ticket, once review is done.

(Why is this here: because I officially deprecated the usage of `client.backend()` and the `forbid` then refuses to compile this file. However the deprecation, nor the changes needed in this file are strictly part of this PR and can be done as a nice starter-ticket)

Reply:
Ticket now at https://github.com/paritytech/substrate/issues/2532